Makefile

```diff
@@ -146,26 +146,34 @@ publish-compile: docker-build-compile
 	docker push $(COMPILE_IMAGE)
 
 docker-clean-worker:
+	rm -rf docker_build_worker.tmpdir
 	docker stop worker_compile || true
 	docker rm worker_compile || true
 
 docker-build-worker: docker-clean-worker
 	docker run \
+		--env=CALLING_USER_ID=$$(id -u $$USER) \
+		--env=DOCKER_GROUP_ID=$$(getent group docker | cut -d: -f3) \
```

Comment:
this doesn't work because macOS doesn't have `getent`

Reply:
OK, thanks! I just changed this to avoid `getent`. The commands work on our office mac, though it doesn't have docker installed, so I can't test the full build on it.

I'll go down the path of setting up our office mac to build Pachyderm, but if you got the chance to pull the new commit and try it, that would be super helpful.